_api_app/app/Shared/Helpers.php

```diff
@@ -270,4 +272,53 @@ public static function asList($val)
 
         return $val;
     }
+
+    public static function uuid_v4()
+    {
+        return sprintf(
+            '%04x%04x-%04x-%04x-%04x-%04x%04x%04x',
+
+        // 32 bits for "time_low"
+            mt_rand(0, 0xffff),
+            mt_rand(0, 0xffff),
+
+        // 16 bits for "time_mid"
+            mt_rand(0, 0xffff),
+
+        // 16 bits for "time_hi_and_version",
+        // four most significant bits holds version number 4
+            mt_rand(0, 0x0fff) | 0x4000,
+
+        // 16 bits, 8 bits for "clk_seq_hi_res",
+        // 8 bits for "clk_seq_low",
+        // two most significant bits holds zero and one for variant DCE1.1
+            mt_rand(0, 0x3fff) | 0x8000,
+
+        // 48 bits for "node"
+            mt_rand(0, 0xffff),
+            mt_rand(0, 0xffff),
+            mt_rand(0, 0xffff)
+        );
+    }
+
+    public static function validate_token($token)
+    {
+        try {
+            $app_key = env('APP_KEY', 'SuperSecretAppKey');
```

Comment:
I think this configuration should go in `config/app.php` file. So it would be the same in all the berta, if we need to use these keys again. Only we need to check what happens on the generation of the key, because it may not update the config automatically in the same request.